cypress/objects/timeLoginPage.js

```diff
@@ -0,0 +1,38 @@
+class TimeLoginPage {
+    visit() {
+        cy.visit('/')
+    }
+
+    getURL() {
+        return cy.url()
```

Comment:
there is no point in this function, as it use another function with no variables, it would make sense to validate URL for this page object like:
`
validateUrl() {
      cy.url().should('include', '/time-logging')
    }
`